### PR TITLE
feat: implement `recursive` layout

### PR DESCRIPTION
**Pull request by TAdev0**
This PR implements the `recursive` layout which includes the following builtins:
- output
- default rangecheck
- pedersen with 128 ratio
- bitwise with 8 ratio

### CODE REVIEW

File: pkg/vm/builtins/layouts.go [hunk 1]
```diff
@@ -37,6 +37,15 @@ func getPlainLayout() Layout {
 	return Layout{Name: "plain", RcUnits: 16, Builtins: []LayoutBuiltin{}}
 }
 
+func getRecursiveLayout() Layout {
+	return Layout{Name: "recursive", RcUnits: 16, Builtins: []LayoutBuiltin{
```
Comment (har777) [inline pkg/vm/builtins/layouts.go:41]: `RcUnits` seems to be `4` in the lambdaclass vm 🤔

Comment (TAdev0) [reply pkg/vm/builtins/layouts.go:41]: true! my bad, will update this

**Comment by TAdev0** [comment]
@har777  ready for review

**Review by har777 — APPROVED** [review]
lgtm.

**Pull request merged**